requirements/pip.txt

```diff
@@ -0,0 +1,2 @@
+pip==20.2.3
+wheel
```

Comment:
This should be pinned too, and setuptools should also be pinned to a particular version here.